sdk/cosmosdb/cosmos/src/utils/logger.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+* Cosmos Diagnostics used for all clients within the Cosmos package
+*/
+const cosmosDiagnostics = require('@azure/logger');
+export const cosmosDiagnosticsLogger = cosmosDiagnostics.createClientLogger("Cosmos Diagnostics");
+
+/**
+ * The log levels supported by the Cosmos Diagnostics.
+ * The log levels are:
+ * - verbose
+ * - DataPlaneRequests - not implemented
+ * - QueryRuntimeStatistics - not implementedv
+ * - PartitionKeyStatistics - not implemented
+ * - ControlPlaneRequests - not implemented
+ */
+export function cosmosDiagnosticsToString(){
+  cosmosDiagnosticsLogger.verbose.bind(cosmosDiagnostics)
+  return String(cosmosDiagnosticsLogger.verbose.bind(cosmosDiagnostics));
+}
```

Comment:
What is this meant to do? The first line of this function does not do anything. `bind` returns a new function that has its arguments bound to specific values. It does not alter `cosmosDiagnosticsLogger.verbose` in any way. The second line converts a bound function into a string, which is the same as calling `.toString()` on it. This will be some string representation of the function's source code. I ran this in the Node REPL:

```
> const logger = require("@azure/logger");
undefined
> const log = logger.createClientLogger("test")
undefined
> String(log.verbose.bind(logger))
'function () { [native code] }'
```